media/json-jackson/pom.xml

```diff
@@ -137,8 +137,8 @@
                     <artifactId>jakarta.activation-api</artifactId>
                 </exclusion>
             </exclusions>
-            <optional>true</optional>
-            <scope>provided</scope>
+<!--            <optional>true</optional>-->
+<!--            <scope>provided</scope>-->
```

Comment:
after updating to the Jackson 2.18.0 (or better saying fixing default constructor to the 

```
protected ProviderBase(MAPPER_CONFIG mconfig,
            LookupCache<AnnotationBundleKey, EP_CONFIG> readerCache,
            LookupCache<AnnotationBundleKey, EP_CONFIG> writerCache)
```

 ), this module becomes required, otherwise, tests fail for 
```
-pl :e2e-server -Dtest=BeanParamExceptionTest
```
with ``java.lang.NoClassDefFoundError: com/fasterxml/jackson/module/jaxb/JaxbAnnotationIntrospector`` at least the phase  ``surefire:3.3.1:test (WithHK2) @ e2e-server``